react-bootstrap/index.d.ts

```diff
@@ -528,6 +528,7 @@ declare namespace ReactBootstrap {
 
     // <Navbar.Toggle />
     interface NavbarToggleProps {
+        onClick?: React.MouseEventHandler<{}>;
```

Comment:
judging by https://github.com/react-bootstrap/react-bootstrap/blob/master/src/NavbarToggle.js, this should be `React.MouseEventHandler<HTMLButtonElement>`.